urllib3/__init__.py

```diff
@@ -18,7 +18,8 @@
 from .util.url import get_host
 from .util.timeout import Timeout
 from .util.retry import Retry
-
+from .util.context import CookieJar, DefaultCookiePolicy, Context
```

Comment:
I say we skip importing `CookieJar` and `DefaultCookiePolicy` into the root urllib3 namespace now that `Context` works by default without specifying them.

Let's add an example in the docs for how to do that, though (provide your own CookieJar and policy).